### ISSUE TITLE
Simulate mobile CPU

### ISSUE THREAD

**Issue by pankajparashar** [MicrosoftDocs/edge-developer#3027]
**Link to the documentation page**:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/#simulate-a-mobile-cpu

**Description of the issue**:
Reading the docs on how to simulate mobile CPU, it's not clear why **Step 2** (_Select the Screenshots checkbox_) is required. I was able to test the [demo page](https://microsoftedge.github.io/Demos/devtools-performance-get-started/) to slow the down the CPU (4x/6x) successfully without needing Step 2.

I think the text below the screenshot which says (_The demo doesn't work well with 6x slowdown, so it just uses 4x slowdown for instructional purposes._) may also be not relevant anymore, as I was able to throttle the CPU to 6x slower

**Screenshots**:
<img width="838" alt="image" src="https://github.com/MicrosoftDocs/edge-developer/assets/38640616/ca241d45-3d6c-4461-ac23-6709f066d26e">

**Microsoft Edge version**:
Version 120.0.2210.144 (Official build) (arm64)

**Comment by captainbrosset** [comment]
Thanks for filing @pankajparashar. Adding to our backlog to fix.

**Comment by captainbrosset** [comment]
Yes that would be great! While I haven't tested yet, I agree with your observations and would welcome a PR on this.
